pkg/sql/ambiguous_commit_test.go

```diff
@@ -1,183 +0,0 @@
-// Copyright 2016 The Cockroach Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-// implied. See the License for the specific language governing
-// permissions and limitations under the License.
-//
-// Author: Spencer Kimball (spencer@cockroachlabs.com)
-
-package sql_test
-
-import (
-	"bytes"
-	"fmt"
-	"sync/atomic"
-	"testing"
-	"time"
-
-	"golang.org/x/net/context"
-
-	"github.com/cockroachdb/cockroach/pkg/base"
-	"github.com/cockroachdb/cockroach/pkg/keys"
-	"github.com/cockroachdb/cockroach/pkg/roachpb"
-	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
-	"github.com/cockroachdb/cockroach/pkg/storage"
-	"github.com/cockroachdb/cockroach/pkg/testutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/testcluster"
-	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
-	"github.com/lib/pq"
-)
-
-// TestAmbiguousCommitDueToLeadershipChange verifies that an ambiguous
```

Comment:
Definitely do not remove this test! This circumstance still is vitally important to solve for (it's the condition behind our original failure on the Jepsen tests).